packages/webpack/src/runtime/runtimeBuilder.js

```diff
@@ -0,0 +1,156 @@
+const { RUNTIME_KEY } = require('../ENUM.json')
+const diag = require('../buildtime/diagnostics.js')
+const { assembleRuntime } = require('./assemble.js')
+const path = require('node:path')
+
+/** @import {LavaMoatPluginOptions} from '../buildtime/types' */
+
+module.exports = {
+  /**
+   * Builds the LavaMoat runtime configuration and generates runtime source code
+   *
+   * @param {Object} params - The parameters object
+   * @param {LavaMoatPluginOptions} params.options - Runtime configuration
+   *   options
+   */
+  runtimeBuilder({ options }) {
+    const runtimeOptions = {
+      scuttleGlobalThis: options.scuttleGlobalThis,
+      lockdown: options.lockdown,
+    }
+
+    return {
+      /**
+       * Generates the LavaMoat runtime source code based on chunk configuration
+       *
+       * @param {Object} params - The parameters object
+       * @param {string | undefined} params.currentChunkName - The webpack chunk
+       *   object
+       * @param {(string | number)[]} params.chunkIds - Array of chunk
+       *   identifiers
+       * @param {Object} params.policyData - LavaMoat security policy
```

Comment:
```suggestion
       * @param {LavaMoatPolicy} params.policyData - LavaMoat security policy
```

idk if that's right tho